### PR TITLE
Add credit file kyc endpoint

### PR DESCRIPTION
**Pull request by dara123**

### CODE REVIEW

File: openapi/components/schemas/CreditFileKyc.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    description: The customer identifier string.
+    allOf:
+      - $ref: ./ResourceId.yaml
+  creditBureau:
+    description: The credit bureau which is used to get credit file data.
+    type: string
+    readOnly: true
+    enum:
+      - equifax
+      - trans-union
```
Comment (adamaltman) [inline openapi/components/schemas/CreditFileKyc.yaml:18]:
```suggestion
      - equifax
      - trans-union
      - experian
```

File: openapi/components/schemas/CreditFileKyc.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+type: object
+properties:
```
Comment (adamaltman) [inline openapi/components/schemas/CreditFileKyc.yaml:2]: missing the `createdTime`.

File: openapi/paths/credit-file-kyc.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
+get:
+  x-products:
+    - Full
+    - Risk Management
+    - Core
+  tags:
+    - AML
+  summary: Retrieve a list of credit file method kyc.
```
Comment (adamaltman) [inline openapi/paths/credit-file-kyc.yaml:8]:
```suggestion
  summary: List credit file KYC reports
```

File: openapi/paths/credit-file-kyc.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  operationId: GetCreditFileKyc
+  x-sdk-operation-name: getCreditFileKyc
+  description: |
+    Retrieve a list of credit file kyc method.
```
Comment (adamaltman) [inline openapi/paths/credit-file-kyc.yaml:12]:
```suggestion
    Retrieve a list of credit file KYC reports.
```

File: openapi/paths/credit-file-kyc.yaml [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  responses:
+    '200':
+      description: Credit file kyc was retrieved successfully.
```
Comment (adamaltman) [inline openapi/paths/credit-file-kyc.yaml:15]:
```suggestion
      description: Credit file KYC report was retrieved successfully.
```

File: openapi/paths/credit-file-kyc.yaml [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+    '404':
+      $ref: ../components/responses/NotFound.yaml
+post:
+  x-products:
+    - Full
+    - Risk Management
+    - Core
+  tags:
+    - AML
+  summary: Create credit file method kyc.
```
Comment (adamaltman) [inline openapi/paths/credit-file-kyc.yaml:33]:
```suggestion
  summary: Run credit file KYC report
```

File: openapi/paths/credit-file-kyc.yaml [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+  operationId: PostCreditFileKyc
+  x-sdk-operation-name: postCreditFileKyc
+  description: |
+    Trigger an API call to credit bureau and store the response.
```
Comment (adamaltman) [inline openapi/paths/credit-file-kyc.yaml:37]:
```suggestion
    Trigger an API call to credit bureau to do a credit file KYC check and store the response.
```

File: openapi/paths/credit-file-kyc.yaml [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+  requestBody:
+    $ref: ../components/requestBodies/CreditFileKyc.yaml
+  responses:
+    '200':
+      description: Credit file kyc was created.
```
Comment (adamaltman) [inline openapi/paths/credit-file-kyc.yaml:42]:
```suggestion
      description: Credit file KYC report was created.
```

**Review by adamaltman — CHANGES_REQUESTED** [review]

File: openapi/components/schemas/CreditFileKyc.yaml [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+  isCivicNumberMatched:
+    description: Return true if address civic number is matched otherwise false.
+    type: boolean
+    readOnly: true
+  isStreetMatched:
+    description: Return true if street address is matched otherwise false.
+    type: boolean
+    readOnly: true
+  isCityMatched:
+    description: Return true if city address is matched otherwise false.
```
Comment (colinbird) [inline openapi/components/schemas/CreditFileKyc.yaml:40]:
```suggestion
    description: Return true if city is matched otherwise false.
```

**Comment by dara123** [comment]
We don't need this new endpoint. We need to update our `PostKycRequest` endpoint to add new subtype `credit-file-proof` and trigger credit file check and `embeded` in the response

**Comment by dara123** [comment]
https://github.com/Rebilly/api-definitions/pull/772

**Pull request closed without merging**